tests/unittest_builder.py

```diff
@@ -191,6 +191,114 @@ class C:
             assert c.fromlineno == 13
         assert c.tolineno == 14
 
+    @staticmethod
+    def test_class_with_docstring() -> None:
+        """Test class nodes which only have docstrings."""
+        code = textwrap.dedent(
+            '''
+        class A:
+            """My docstring"""
+            var = 1
+
+        class B:
+            """My docstring"""
+
+        class C:
+            """My docstring
+            is long."""
+
+        class D:
+            """My docstring
+            is long.
+            """
+
+        class E:
+            ...
+        '''
+        )
+
+        ast_module: nodes.Module = builder.parse(code)  # type: ignore[assignment]
```

Comment:
```suggestion
        ast_module = builder.parse(code)
```
Why is this necessary? `builder.parse(...)` is annotated to return a `nodes.Module` instance.

Reply:
Copied from the test above 😉